docs/administration/projects/project-motd.md

```diff
@@ -29,13 +32,13 @@ rd projects configure set -p MyProject -- \
     PUT /api/13/project/myproject/motd.md
     Content-Type: text/plain
 
-    This is the **motd** for MyProject
+    "This is the **motd** for MyProject"
 
 ## Filesystem
 
 If using the _filesystem_ storage type only, you can create the file in the project base directory:
```

Comment:
Let's add context here as well as to "why" they might consider this.  If it's a rare use case we can mention that as well.

Overall let's guide the user to picking the best path for managing this.  (Probably GUI?)